components/browser/session/src/main/java/mozilla/components/browser/session/storage/DefaultSessionStorage.kt

```diff
@@ -109,7 +109,7 @@ class DefaultSessionStorage(
         return try {
             val json = JSONObject()
             json.put(VERSION_KEY, VERSION)
-            json.put(SELECTED_SESSION_KEY, sessionManager.selectedSession.id)
+            json.put(SELECTED_SESSION_KEY, sessionManager.selectedSession?.id)
```

Comment:
I think we should put an empty string in here instead of "null". When this value gets restored we're looking up a session with its ID using a non-nullable String.

Reply:
👍